components/resume/company/company.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react'
+import Position from './positions';
+import Skills from './skills';
+
+interface Position {
+    title: string;
+    startDate: string;
+    endDate?: string;
+}
+
+interface CompanyProps {
+    name: string;
+    positions: Position[];
+    startDate: string;
+    endDate?: string;
+    skills: string[];
+    highlights: string[]
+}
+
+const Company: FC<CompanyProps> = ({ name, skills, positions, highlights }) => {
+    return <div>
+        <h3>{name}</h3>
+        <Skills skills={skills} />
+        <div>
+            {positions.map(position =>
```

Comment:
For now keeping the iteration of positions and highlights here and passing off the details to a dedicated component, mainly because I want to limit passing around "god" objects to sub components. I considered making a very shallow <Positions> component which would just do this iteration and pass off details to a Position component, but for now holding off on it. 

Maybe I'll take that route if I see more list specific styles that I want to add vs list item specific.